llmproxy/tests/unit_tests/models/test_llama2.py

```diff
@@ -0,0 +1,33 @@
+import os
+from llmproxy.models.llama2 import Llama2, Llama2Model
+from dotenv import load_dotenv
+
+load_dotenv()
+api_key = os.getenv("LLAMA2_API_KEY")
+
+def test_llama2_constructor() -> None:
+    llama2_Con = Llama2()
+    assert llama2_Con.api_key == ""
+
+def test_llama2_prompt() -> None:
+    llama2_emp_promt = ""
+    output = Llama2(prompt=llama2_emp_promt)
+    assert output.prompt == ""
```

Comment:
I am not sure you can do this kind of test with the ream Llama2 instance. You will need to mock it, or else unit test are going to cost tokens when we pay for the LLMs